perun-base/src/main/java/cz/metacentrum/perun/core/api/BeansUtils.java

```diff
@@ -810,6 +810,12 @@ public static JavaMailSender getDefaultMailSender() {
 			mailProps.setProperty("mail.smtp.starttls.enable", String.valueOf(BeansUtils.getCoreConfig().isSmtpStartTls()));
 			mailProps.setProperty("mail.debug", String.valueOf(BeansUtils.getCoreConfig().isMailDebug()));
 
+			// set 'smtpFrom' property only if it is set in perun.properties
+			String smtpFrom = BeansUtils.getCoreConfig().getSmtpFrom();
+			if (smtpFrom != null && !smtpFrom.isEmpty()) {
```

Comment:
I would rather use this one. :eyes: 
```suggestion
			if (smtpFrom != null && !smtpFrom.isBlank()) {
```

Reply:
Yeah that makes sense :+1: